### ISSUE TITLE
Broken database triggers at OC PL

### ISSUE THREAD

**Issue by following5** [opencaching/opencaching-pl#1939]
```
okapi\core\Exception\DbException:
SQL Error 1449: The user specified as a definer ('root'@'******') does not exist

The query was:

            insert into cache_logs (
                uuid, cache_id, user_id, type, date, text, text_html, text_htmledit,
                last_modified, date_created, node
            ) values (
                '3324519c-9878-4bbb-acea-7e4b8d741991',
                '62954',
                '87321',
```

Looks like 100_init DB update has been run with
* $opt['db']['admin_username'] = 'root';

and then the 'root' user was deleted or renamed ...?

**Comment by deg-pl** [comment]
@following5 Yes - root was deleted and all triggers and stored procedures has changed definer to current OC PL DB user.
But something is strange - we have no $opt['db']['admin_username'] in config! So scripts should be runned as default db user, not root.

**Comment by following5** [comment]
*Did* 100_init run at OC PL? Did someone temporarily define `$opt['db']['admin_username'] = 'root'` to run it?

 I cannot see if it was run at OC PL. Only noticed by okapi/devel/dbstruct that exactly those triggers had 'root' definer which are update by 100_init.

**Comment by following5** [comment]
Anyway, this was a temporary problem. Everything not defined by OC user, and that's the same uses that will run any DB updates.

**Comment by andrixnet** [comment]
Can you please document the purpose and requirements for:
`$opt['db']['admin_username']`
